dash_cytoscape/Cytoscape.py

```diff
@@ -104,7 +107,8 @@ class attribute).
 - autounselectify (boolean; optional): Whether nodes should be unselectified (immutable selection state) by
 default (if true, overrides individual element state).
 - autoRefreshLayout (boolean; optional): Whether the layout should be refreshed when elements are added or removed.
-- tapNode (dict; optional): The complete node dictionary returned when you tap or click it.
+- tapNode (dict; optional): The complete node dictionary returned when you tap or click it. Cannot be modified by a
```

Comment:
Maybe say that it's "read-only" instead -- that might be clearer!

Reply:
Done